contracts/crowdsale/Crowdsale.sol

```diff
@@ -59,12 +59,16 @@ contract Crowdsale {
     return new MintableToken();
   }
 
-
   // fallback function can be used to buy tokens
   function () external payable {
     buyTokens(msg.sender);
   }
 
+  // Override this function to create logic for periodization
+  function getTokenAmount(uint256 weiAmount) internal constant returns(uint256) {
```

Comment:
Can we change `constant` for `view`? They're aliases but we use `view` throughout the codebase.

Reply:
good catch!